crates/ty_python_semantic/resources/mdtest/type_qualifiers/final.md

```diff
@@ -88,6 +88,8 @@ class C:
         self.FINAL_C: Final[int] = 1
         self.FINAL_D: Final = 1
         self.FINAL_E: Final
+        # TODO: Should not be an error
+        # error: [invalid-assignment] "Cannot assign to final attribute `FINAL_E` on type `Self@__init__`"
```

Comment:
This (and below) was really only "working" because we didn't infer a type of `self`, not because we ever added support for this. As far as I remember, no other type checker supports this either and we decided at the time that it was not important to implement.